src/sidebar/components/datalist.js

```diff
@@ -0,0 +1,106 @@
+import { createElement } from 'preact';
+import classnames from 'classnames';
+import { useMemo } from 'preact/hooks';
+import propTypes from 'prop-types';
+
+const defaultListFormatter = item => item;
+
+/**
+ * Custom datalist component. Use this in conjunction with an <input> field.
+ * To make this component W3 accessibility compliant, it is is intended to be
```

Comment:
Ah, I wasn't clear enough in my previous comment. Can you add a mention of the name of the ARIA design pattern the datalist + input field together implement (is it "ARIA 1.1 Combobox with listbox popup"?)

Reply:
awe! thx